### PR TITLE
Mayor's Major Balance

### PR DESCRIPTION
**Pull request by Optimism333**
## About The Pull Request
Adjusts the stats of mayor to be less insane, and scales their stats and combat skills with age (now that we have middle-aged mayors).

Also added a bit more money to their bank account based on a request: 50 -> 80

### Before: ###
image

### After: ###
image

## Why It's Good For The Game

The middle-aged change for mayor without any balance adjustments had unforeseen consequences.

This attempts to address the consequences.

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

File: code/modules/jobs/job_types/roguetown/garrison/villagechief.dm [hunk 1]
```diff
@@ -49,9 +49,15 @@
...
-		H.change_stat("endurance", 4)
-		H.change_stat("speed", 3)
-		H.change_stat("intelligence", 5)
+		if(H.age == AGE_OLD)
+			H.mind.adjust_skillrank(/datum/skill/combat/polearms, 1, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/combat/axesmaces, 1, TRUE)
+			H.change_stat("strength", 3)
+			H.change_stat("perception", 2)
+			H.change_stat("endurance", 1)
+			H.change_stat("intelligence", 3)
```
Comment (ThePainkiller) [inline code/modules/jobs/job_types/roguetown/garrison/villagechief.dm:58]:
```suggestion
			H.change_stat("intelligence", 1)
```
Old people already get +2 INT and that's enough, even with a 20% penalty on learning boons this is INSANE even by job description standards, so at least +3 INT total sounds good for "wise beyond your years" but is by no way justification to make them wizard 2, learner of every job under creation

**Review by ThePainkiller — CHANGES_REQUESTED** [review]
No combat-tier skills for non combat classes, as always. Leaving them at 2 is a compromise much to my chagrin, but it shouldn't make them overpowered nor as combat ready as a town guard but insanely superior through stats.

**Comment by Optimism333** [comment]
I can't say I agree with all the changes, but I think they are all applied now.

**Review by ThePainkiller — APPROVED** [review]

**Comment by ThePainkiller** [comment]
Now this here's a proper nerf my god, can't believe he was stronger than a royal guard

**Pull request merged**